02-microarray/00-intro-to-microarray.Rmd

```diff
@@ -6,3 +6,60 @@ output:
     toc: true
     toc_float: true
 ---
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Introduction to Microarray technology](#introduction-to-microarray-technology)
+  - [Microarray data **strengths**:](#microarray-data-strengths)
+  - [Microarray data **limitations**:](#microarray-data-limitations)
+- [About quantile normalization](#about-quantile-normalization)
+- [More resources on microarray technology:](#more-resources-on-microarray-technology)
+  - [Why doesn't the gene I care about show up in this microarray dataset?](#why-doesnt-the-gene-i-care-about-show-up-in-this-microarray-dataset)
+- [References](#references)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Introduction to Microarray technology
+
+Data analyses are generally not "one size fits all"; this is particularly true between RNA-seq vs microarray data.
+This tutorial has example analyses [organized by technology](../01-getting-started/getting-started.html#about-how-this-tutorial-book-is-structured) so you can follow examples that are more closely tailored to the nature of the data at hand. 
+
+As with all experimental methods, microarray has strengths and limitations that you should consider in regards to your scientific questions. 
+
+### Microarray data **strengths**:  
+
+- Less expensive than RNA-seq - can afford more replicates and get higher statistical power [@Tarca2006]. 
+- Microarray has a faster turn-around than RNA-seq [@LCSciences2014].
+
+### Microarray data **limitations**:  
+
+- If a transcript doesn't have a probe designed to it, you won't get it measured; can't be used for transcript discovery [@Mantione2014]. 
```

Comment:
My question -- to what extent does DNA variation impact a translated transcript's ability to hybridize, and how does that compare to an alignment tool's difficulty to align transcripts that don't match the reference perfectly, i.e. which  approach is more robust to variation? my intuition is RNA-seq but could definitely be off base

Reply:
The extent to which a SNP affects microarray hybridization and its resulting fluorescence signal probably depends on a number of things like how long the probes are, where in the probe the SNP occurs, whether the SNP is homozygous, how pervasive the SNP is in the tissues and organism. And I don't know how well these things can be accounted for without also doing a carefully designed SNP array side by side with the the gene expression array. 

I see there are [some tools](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1933513/) that try to detect SNPs from gene expression microarray,  but I don't get the impression that they used very widespread-ly(?) and I don't think the [tools we use in refine.bio (e.g. SCAN)](http://docs.refine.bio/en/latest/main_text.html#processing-information) address this for microarray data (could be wrong on either of these accounts). 

@jaclyn-taroni may have a more informed opinion to add about this. 

The second follow up question based on this context is whether this is something our users will want to know about and I am suspecting no? (also could be wrong).

Reply:
I agree -- I think it's not important for the target audience of this into... just my musing 🤔

Reply:
A huge part of my postdoc... gone by the wayside with cheap sequencing.